templates/client-snapshot-agent-deployment.yaml

```diff
@@ -132,6 +142,29 @@ spec:
           - name: aclconfig
             mountPath: /consul/aclconfig
       {{- end }}
+      {{- if (and .Values.global.tls.enabled .Values.global.tls.enableAutoEncrypt) }}
+      - name: get-consul-client-ca
+        image: {{ .Values.global.imageK8S }}
+        command:
+          - "/bin/sh"
+          - "-ec"
```

Comment:
I thought `-e` will make the pod exit immediately if any of the commands fail. Although, it's also not strictly needed here because there's only one command.